src/jobs/helm_promote.yaml

```diff
@@ -1,8 +1,13 @@
 description: "Set version and publish to chart museum"
 
 parameters:
-  version:
-    description: "Name of the helm chart"
+  source_version:
+    description: "Name of the source helm chart"
```

Comment:
This is very confusing. What do you mean by source version? What does version have to do with name of the chart?

Reply:
Its a helm_promote step. We have a source and a target.
Downloads the source and uploads to the target

Reply:
Do you have a suggestion of better names?

Reply:
I wrote more in the other comment. Please check.
After the extra context my main feedback is that you created a name `promote` that does not exist in helm and when you do that you should document it really well for people to use it consciously and know what are the effects bellow.